content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -55,12 +55,15 @@ There are two types of hook handlers that can be implemented for Containers:
 * Exec - Executes a specific command, such as `pre-stop.sh`, inside the cgroups and namespaces of the Container.
 Resources consumed by the command are counted against the Container.
 * HTTP - Executes an HTTP request against a specific endpoint on the Container.
+* Sleep - Executes a sleep command to pause a container for a specified duration. 
```

Comment:
Can we avoid "executes" which implies that `sleep` needs to be a binary in the image?

How about "Sleep - Pauses for a specified duration."

Reply:
Updated to `Sleep - Pauses the container for a specified duration. `